statsmodels/discrete/discrete_model.py

```diff
@@ -1882,8 +2026,36 @@ def score_obs(self, params):
 
         y = self.endog
         X = self.exog
-        L = self.cdf(np.dot(X, params))
-        return (y - L)[:,None] * X
+        fitted = self.predict(params)
+        return (y - fitted)[:,None] * X
+
+    def score_factor(self, params):
+        """
+        Logit model derivative of the log-likelihood with respect to linpred
+
+        Parameters
+        ----------
+        params: array-like
+            The parameters of the model
+
+        Returns
+        -------
+        jac : array-like
```

Comment:
is `jac` standard here?  jac_factor?

Reply:
docstring is still wrong, i.e. is for score and not score_factor

jac was the old name that we/I used at the beginning of statsmodels